sdk/purview/purview-scanning-rest/README.md

```diff
@@ -0,0 +1,51 @@
+# Azure Purview Scanning Rest-Level client library for JavaScript
+
+## Getting started
+
+### Currently supported environments
+
+- Node.js version 13.x.x or higher
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+
+### Install the `@azure-rest/purview-scanning` package
+
+Install the Azure Purview Scanning client library for JavaScript with `npm`:
+
+```bash
+npm install @azure-rest/purview-scanning
+```
+
+### Create and authenticate a `PurviewScanning`
+
+## Key concepts
+
+## Examples
+
```

Comment:
Please do not forget to fill in these sections.

Reply:
Done!